README.md

```diff
@@ -25,8 +27,8 @@ This project has the following dependencies:
 
 The following steps will assume the following
 
-- The instructions will be based off of a common base folder: $DEMO_BASE. For the steps below, we will use DEMO_BASE of ~/ for simplicty. 
-- This current project has been fetched to $DEMO_BASE
+- The instructions will be based off of a common base folder: `~/`. For the steps below, we will use `~/` for simplicity. 
```

Comment:
This line is incorrect

Reply:
It is ok to show commands with $Demo_BASE and prompt with a particular directory (here '~/')